src/vs/workbench/contrib/webview/browser/themeing.ts

```diff
@@ -75,7 +76,7 @@ export class WebviewThemeDataProvider extends Disposable {
 			};
 
 			const activeTheme = ApiThemeClassName.fromTheme(theme);
-			this._cachedWebViewThemeData = { styles, activeTheme, themeLabel: theme.label, };
+			this._cachedWebViewThemeData = { styles, activeTheme, themeLabel: theme.label, themeId: theme.settingsId };
```

Comment:
@aeschli Can you please confirm that `settingsId` is a good value to use here? I switched to use this since `theme.id` is not friendly to consume. Here's some example values for Dark+:

- `theme.id`: `vs-dark vscode-theme-defaults-themes-dark_plus-json`
- `theme.settingsId: `Default Dark+`

Reply:
Yes, sorry for not realizing this. settingsId is what we want (it's unique and it's what can be used in the settings)